code/modules/research/designs/power_designs.dm

```diff
@@ -29,7 +29,7 @@
 	desc = "A power cell that holds 20 MJ of energy."
 	id = "super_cell"
 	build_type = PROTOLATHE | MECHFAB
-	materials = list(/datum/material/iron = 700, /datum/material/glass = 70, /datum/material/dilithium = 250) //Yogs: added dilithium
+	materials = list(/datum/material/iron = 700, /datum/material/glass = 80)
```

Comment:
so in exchange for 250 dilithium, it now costs... 10 more glass...?
250 of basically bluespare for 10 glass?

that dont sound round

Reply:
/datum/material/iron = 700, /datum/material/gold = 75, /datum/material/silver = 75, /datum/material/glass = 80

so its a cheaper hyper cell

Reply:
Done